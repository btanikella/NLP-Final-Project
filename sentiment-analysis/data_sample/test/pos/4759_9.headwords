born
loved
bought
actor
one
