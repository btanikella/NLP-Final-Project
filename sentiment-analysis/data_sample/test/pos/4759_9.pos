PRP VBD VBN IN CD IN RB JJ JJ NN IN RB IN WP PRP VBD IN NN -LRB- IN DT NN IN DT CD POS -RRB-
PRP RB VBD DT NN -LRB- CC DT JJ NN -RRB- CC MD RB VB TO VB PRP WRB PRP VBD IN -LRB- VBG DT NNP NN NN RB IN NN -RRB-
PRP VBD PRP IN NN DT JJ NNS RB CC VBP VBN PRP RB IN RB CC PRP MD VB  PRP RB VBP PRP IN
NNP NNP VBD DT JJ NN IN DT CD POS -LRB- NNP VBZ CD IN PRP$ JJ NNS  NNP NNP FW -RRB- CC IN DT NNS VBZ RB DT JJ NN  PRP VBZ PRP RB IN DT NN DT NN -LRB- NNP  RB CD POS  IN VBD JJ NN -RRB- CC VBZ PRP DT JJ NNS CC DT NN IN NN IN DT NN IN DT NN
CC RB  DT NN NN VBZ CD IN DT JJ JJS
