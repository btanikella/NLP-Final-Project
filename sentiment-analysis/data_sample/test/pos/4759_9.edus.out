Sentence #1 (28 tokens):
I was born in '68 with not much parental guidance as far as what I watched on TV (as a kid in the 70's).
[Text=I CharacterOffsetBegin=4 CharacterOffsetEnd=5 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=6 CharacterOffsetEnd=9 PartOfSpeech=VBD] [Text=born CharacterOffsetBegin=10 CharacterOffsetEnd=14 PartOfSpeech=VBN] [Text=in CharacterOffsetBegin=15 CharacterOffsetEnd=17 PartOfSpeech=IN] [Text='68 CharacterOffsetBegin=18 CharacterOffsetEnd=21 PartOfSpeech=CD] [Text=with CharacterOffsetBegin=22 CharacterOffsetEnd=26 PartOfSpeech=IN] [Text=not CharacterOffsetBegin=27 CharacterOffsetEnd=30 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=31 CharacterOffsetEnd=35 PartOfSpeech=JJ] [Text=parental CharacterOffsetBegin=36 CharacterOffsetEnd=44 PartOfSpeech=JJ] [Text=guidance CharacterOffsetBegin=45 CharacterOffsetEnd=53 PartOfSpeech=NN] [Text=as CharacterOffsetBegin=54 CharacterOffsetEnd=56 PartOfSpeech=IN] [Text=far CharacterOffsetBegin=57 CharacterOffsetEnd=60 PartOfSpeech=RB] [Text=as CharacterOffsetBegin=61 CharacterOffsetEnd=63 PartOfSpeech=IN] [Text=what CharacterOffsetBegin=64 CharacterOffsetEnd=68 PartOfSpeech=WP] [Text=I CharacterOffsetBegin=69 CharacterOffsetEnd=70 PartOfSpeech=PRP] [Text=watched CharacterOffsetBegin=71 CharacterOffsetEnd=78 PartOfSpeech=VBD] [Text=on CharacterOffsetBegin=79 CharacterOffsetEnd=81 PartOfSpeech=IN] [Text=TV CharacterOffsetBegin=82 CharacterOffsetEnd=84 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=85 CharacterOffsetEnd=86 PartOfSpeech=-LRB-] [Text=as CharacterOffsetBegin=86 CharacterOffsetEnd=88 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=89 CharacterOffsetEnd=90 PartOfSpeech=DT] [Text=kid CharacterOffsetBegin=91 CharacterOffsetEnd=94 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=95 CharacterOffsetEnd=97 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=98 CharacterOffsetEnd=101 PartOfSpeech=DT] [Text=70 CharacterOffsetBegin=102 CharacterOffsetEnd=104 PartOfSpeech=CD] [Text='s CharacterOffsetBegin=104 CharacterOffsetEnd=106 PartOfSpeech=POS] [Text=-RRB- CharacterOffsetBegin=106 CharacterOffsetEnd=107 PartOfSpeech=-RRB-] [Text=. CharacterOffsetBegin=107 CharacterOffsetEnd=108 PartOfSpeech=.] 
root(ROOT-0, born-3)
nsubjpass(born-3, I-1)
auxpass(born-3, was-2)
prep(born-3, in-4)
pobj(in-4, '68-5)
prep(born-3, with-6)
neg(guidance-10, not-7)
advmod(guidance-10, much-8)
amod(guidance-10, parental-9)
pobj(with-6, guidance-10)
advmod(far-12, as-11)
advmod(guidance-10, far-12)
prep(far-12, as-13)
dobj(watched-16, what-14)
nsubj(watched-16, I-15)
pcomp(as-13, watched-16)
prep(watched-16, on-17)
pobj(on-17, TV-18)
punct(as-20, -LRB--19)
prep(TV-18, as-20)
det(kid-22, a-21)
pobj(as-20, kid-22)
prep(kid-22, in-23)
det(70-25, the-24)
pobj(in-23, 70-25)
possessive(70-25, 's-26)
punct(as-20, -RRB--27)
punct(born-3, .-28)

Sentence #2 (33 tokens):
I always loved this movie (and the French Connection) and would always try to catch it whenever it was on (checking the Sunday TV guide ahead of time).
[Text=I CharacterOffsetBegin=113 CharacterOffsetEnd=114 PartOfSpeech=PRP] [Text=always CharacterOffsetBegin=115 CharacterOffsetEnd=121 PartOfSpeech=RB] [Text=loved CharacterOffsetBegin=122 CharacterOffsetEnd=127 PartOfSpeech=VBD] [Text=this CharacterOffsetBegin=128 CharacterOffsetEnd=132 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=133 CharacterOffsetEnd=138 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=139 CharacterOffsetEnd=140 PartOfSpeech=-LRB-] [Text=and CharacterOffsetBegin=140 CharacterOffsetEnd=143 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=144 CharacterOffsetEnd=147 PartOfSpeech=DT] [Text=French CharacterOffsetBegin=148 CharacterOffsetEnd=154 PartOfSpeech=JJ] [Text=Connection CharacterOffsetBegin=155 CharacterOffsetEnd=165 PartOfSpeech=NN] [Text=-RRB- CharacterOffsetBegin=165 CharacterOffsetEnd=166 PartOfSpeech=-RRB-] [Text=and CharacterOffsetBegin=167 CharacterOffsetEnd=170 PartOfSpeech=CC] [Text=would CharacterOffsetBegin=171 CharacterOffsetEnd=176 PartOfSpeech=MD] [Text=always CharacterOffsetBegin=177 CharacterOffsetEnd=183 PartOfSpeech=RB] [Text=try CharacterOffsetBegin=184 CharacterOffsetEnd=187 PartOfSpeech=VB] [Text=to CharacterOffsetBegin=188 CharacterOffsetEnd=190 PartOfSpeech=TO] [Text=catch CharacterOffsetBegin=191 CharacterOffsetEnd=196 PartOfSpeech=VB] [Text=it CharacterOffsetBegin=197 CharacterOffsetEnd=199 PartOfSpeech=PRP] [Text=whenever CharacterOffsetBegin=200 CharacterOffsetEnd=208 PartOfSpeech=WRB] [Text=it CharacterOffsetBegin=209 CharacterOffsetEnd=211 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=212 CharacterOffsetEnd=215 PartOfSpeech=VBD] [Text=on CharacterOffsetBegin=216 CharacterOffsetEnd=218 PartOfSpeech=IN] [Text=-LRB- CharacterOffsetBegin=219 CharacterOffsetEnd=220 PartOfSpeech=-LRB-] [Text=checking CharacterOffsetBegin=220 CharacterOffsetEnd=228 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=229 CharacterOffsetEnd=232 PartOfSpeech=DT] [Text=Sunday CharacterOffsetBegin=233 CharacterOffsetEnd=239 PartOfSpeech=NNP] [Text=TV CharacterOffsetBegin=240 CharacterOffsetEnd=242 PartOfSpeech=NN] [Text=guide CharacterOffsetBegin=243 CharacterOffsetEnd=248 PartOfSpeech=NN] [Text=ahead CharacterOffsetBegin=249 CharacterOffsetEnd=254 PartOfSpeech=RB] [Text=of CharacterOffsetBegin=255 CharacterOffsetEnd=257 PartOfSpeech=IN] [Text=time CharacterOffsetBegin=258 CharacterOffsetEnd=262 PartOfSpeech=NN] [Text=-RRB- CharacterOffsetBegin=262 CharacterOffsetEnd=263 PartOfSpeech=-RRB-] [Text=. CharacterOffsetBegin=263 CharacterOffsetEnd=264 PartOfSpeech=.] 
root(ROOT-0, loved-3)
nsubj(loved-3, I-1)
advmod(loved-3, always-2)
det(movie-5, this-4)
dobj(loved-3, movie-5)
punct(Connection-10, -LRB--6)
cc(Connection-10, and-7)
det(Connection-10, the-8)
amod(Connection-10, French-9)
dep(movie-5, Connection-10)
punct(Connection-10, -RRB--11)
cc(loved-3, and-12)
aux(try-15, would-13)
advmod(try-15, always-14)
conj(loved-3, try-15)
aux(catch-17, to-16)
xcomp(try-15, catch-17)
dobj(catch-17, it-18)
advmod(was-21, whenever-19)
nsubj(was-21, it-20)
advcl(catch-17, was-21)
prep(was-21, on-22)
punct(checking-24, -LRB--23)
pcomp(on-22, checking-24)
det(guide-28, the-25)
nn(guide-28, Sunday-26)
nn(guide-28, TV-27)
dobj(checking-24, guide-28)
advmod(guide-28, ahead-29)
prep(ahead-29, of-30)
pobj(of-30, time-31)
punct(checking-24, -RRB--32)
punct(loved-3, .-33)

Sentence #3 (26 tokens):
I bought it on DVD a few years ago and have watched it twice since then and I must say, I STILL LOVE IT!!!!
[Text=I CharacterOffsetBegin=269 CharacterOffsetEnd=270 PartOfSpeech=PRP] [Text=bought CharacterOffsetBegin=271 CharacterOffsetEnd=277 PartOfSpeech=VBD] [Text=it CharacterOffsetBegin=278 CharacterOffsetEnd=280 PartOfSpeech=PRP] [Text=on CharacterOffsetBegin=281 CharacterOffsetEnd=283 PartOfSpeech=IN] [Text=DVD CharacterOffsetBegin=284 CharacterOffsetEnd=287 PartOfSpeech=NN] [Text=a CharacterOffsetBegin=288 CharacterOffsetEnd=289 PartOfSpeech=DT] [Text=few CharacterOffsetBegin=290 CharacterOffsetEnd=293 PartOfSpeech=JJ] [Text=years CharacterOffsetBegin=294 CharacterOffsetEnd=299 PartOfSpeech=NNS] [Text=ago CharacterOffsetBegin=300 CharacterOffsetEnd=303 PartOfSpeech=RB] [Text=and CharacterOffsetBegin=304 CharacterOffsetEnd=307 PartOfSpeech=CC] [Text=have CharacterOffsetBegin=308 CharacterOffsetEnd=312 PartOfSpeech=VBP] [Text=watched CharacterOffsetBegin=313 CharacterOffsetEnd=320 PartOfSpeech=VBN] [Text=it CharacterOffsetBegin=321 CharacterOffsetEnd=323 PartOfSpeech=PRP] [Text=twice CharacterOffsetBegin=324 CharacterOffsetEnd=329 PartOfSpeech=RB] [Text=since CharacterOffsetBegin=330 CharacterOffsetEnd=335 PartOfSpeech=IN] [Text=then CharacterOffsetBegin=336 CharacterOffsetEnd=340 PartOfSpeech=RB] [Text=and CharacterOffsetBegin=341 CharacterOffsetEnd=344 PartOfSpeech=CC] [Text=I CharacterOffsetBegin=345 CharacterOffsetEnd=346 PartOfSpeech=PRP] [Text=must CharacterOffsetBegin=347 CharacterOffsetEnd=351 PartOfSpeech=MD] [Text=say CharacterOffsetBegin=352 CharacterOffsetEnd=355 PartOfSpeech=VB] [Text=, CharacterOffsetBegin=355 CharacterOffsetEnd=356 PartOfSpeech=,] [Text=I CharacterOffsetBegin=357 CharacterOffsetEnd=358 PartOfSpeech=PRP] [Text=STILL CharacterOffsetBegin=359 CharacterOffsetEnd=364 PartOfSpeech=RB] [Text=LOVE CharacterOffsetBegin=365 CharacterOffsetEnd=369 PartOfSpeech=VBP] [Text=IT CharacterOffsetBegin=370 CharacterOffsetEnd=372 PartOfSpeech=PRP] [Text=!!!! CharacterOffsetBegin=372 CharacterOffsetEnd=376 PartOfSpeech=IN] 
root(ROOT-0, bought-2)
nsubj(bought-2, I-1)
dobj(bought-2, it-3)
prep(bought-2, on-4)
pobj(on-4, DVD-5)
det(years-8, a-6)
amod(years-8, few-7)
npadvmod(ago-9, years-8)
advmod(bought-2, ago-9)
cc(bought-2, and-10)
aux(watched-12, have-11)
conj(bought-2, watched-12)
dobj(watched-12, it-13)
advmod(since-15, twice-14)
prep(watched-12, since-15)
pobj(since-15, then-16)
cc(bought-2, and-17)
nsubj(say-20, I-18)
aux(say-20, must-19)
conj(bought-2, say-20)
punct(bought-2, ,-21)
nsubj(LOVE-24, I-22)
advmod(LOVE-24, STILL-23)
dep(bought-2, LOVE-24)
dobj(LOVE-24, IT-25)
dep(LOVE-24, !!!!-26)

Sentence #4 (72 tokens):
Roy S. was a great actor from the 70's (Jaws is one of my all-time favs, Marathon Man etc) and although the 7-ups is not an Oscar-worthy film, it puts you precisely in a time a place (NY, early 70's, as did French Connection) and gives you some tough characters and a glimpse of life as a cop at that time.
[Text=Roy CharacterOffsetBegin=381 CharacterOffsetEnd=384 PartOfSpeech=NNP] [Text=S. CharacterOffsetBegin=385 CharacterOffsetEnd=387 PartOfSpeech=NNP] [Text=was CharacterOffsetBegin=388 CharacterOffsetEnd=391 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=392 CharacterOffsetEnd=393 PartOfSpeech=DT] [Text=great CharacterOffsetBegin=394 CharacterOffsetEnd=399 PartOfSpeech=JJ] [Text=actor CharacterOffsetBegin=400 CharacterOffsetEnd=405 PartOfSpeech=NN] [Text=from CharacterOffsetBegin=406 CharacterOffsetEnd=410 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=411 CharacterOffsetEnd=414 PartOfSpeech=DT] [Text=70 CharacterOffsetBegin=415 CharacterOffsetEnd=417 PartOfSpeech=CD] [Text='s CharacterOffsetBegin=417 CharacterOffsetEnd=419 PartOfSpeech=POS] [Text=-LRB- CharacterOffsetBegin=420 CharacterOffsetEnd=421 PartOfSpeech=-LRB-] [Text=Jaws CharacterOffsetBegin=421 CharacterOffsetEnd=425 PartOfSpeech=NNP] [Text=is CharacterOffsetBegin=426 CharacterOffsetEnd=428 PartOfSpeech=VBZ] [Text=one CharacterOffsetBegin=429 CharacterOffsetEnd=432 PartOfSpeech=CD] [Text=of CharacterOffsetBegin=433 CharacterOffsetEnd=435 PartOfSpeech=IN] [Text=my CharacterOffsetBegin=436 CharacterOffsetEnd=438 PartOfSpeech=PRP$] [Text=all-time CharacterOffsetBegin=439 CharacterOffsetEnd=447 PartOfSpeech=JJ] [Text=favs CharacterOffsetBegin=448 CharacterOffsetEnd=452 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=452 CharacterOffsetEnd=453 PartOfSpeech=,] [Text=Marathon CharacterOffsetBegin=454 CharacterOffsetEnd=462 PartOfSpeech=NNP] [Text=Man CharacterOffsetBegin=463 CharacterOffsetEnd=466 PartOfSpeech=NNP] [Text=etc CharacterOffsetBegin=467 CharacterOffsetEnd=470 PartOfSpeech=FW] [Text=-RRB- CharacterOffsetBegin=470 CharacterOffsetEnd=471 PartOfSpeech=-RRB-] [Text=and CharacterOffsetBegin=472 CharacterOffsetEnd=475 PartOfSpeech=CC] [Text=although CharacterOffsetBegin=476 CharacterOffsetEnd=484 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=485 CharacterOffsetEnd=488 PartOfSpeech=DT] [Text=7-ups CharacterOffsetBegin=489 CharacterOffsetEnd=494 PartOfSpeech=NNS] [Text=is CharacterOffsetBegin=495 CharacterOffsetEnd=497 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=498 CharacterOffsetEnd=501 PartOfSpeech=RB] [Text=an CharacterOffsetBegin=502 CharacterOffsetEnd=504 PartOfSpeech=DT] [Text=Oscar-worthy CharacterOffsetBegin=505 CharacterOffsetEnd=517 PartOfSpeech=JJ] [Text=film CharacterOffsetBegin=518 CharacterOffsetEnd=522 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=522 CharacterOffsetEnd=523 PartOfSpeech=,] [Text=it CharacterOffsetBegin=524 CharacterOffsetEnd=526 PartOfSpeech=PRP] [Text=puts CharacterOffsetBegin=527 CharacterOffsetEnd=531 PartOfSpeech=VBZ] [Text=you CharacterOffsetBegin=532 CharacterOffsetEnd=535 PartOfSpeech=PRP] [Text=precisely CharacterOffsetBegin=536 CharacterOffsetEnd=545 PartOfSpeech=RB] [Text=in CharacterOffsetBegin=546 CharacterOffsetEnd=548 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=549 CharacterOffsetEnd=550 PartOfSpeech=DT] [Text=time CharacterOffsetBegin=551 CharacterOffsetEnd=555 PartOfSpeech=NN] [Text=a CharacterOffsetBegin=556 CharacterOffsetEnd=557 PartOfSpeech=DT] [Text=place CharacterOffsetBegin=558 CharacterOffsetEnd=563 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=564 CharacterOffsetEnd=565 PartOfSpeech=-LRB-] [Text=NY CharacterOffsetBegin=565 CharacterOffsetEnd=567 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=567 CharacterOffsetEnd=568 PartOfSpeech=,] [Text=early CharacterOffsetBegin=569 CharacterOffsetEnd=574 PartOfSpeech=RB] [Text=70 CharacterOffsetBegin=575 CharacterOffsetEnd=577 PartOfSpeech=CD] [Text='s CharacterOffsetBegin=577 CharacterOffsetEnd=579 PartOfSpeech=POS] [Text=, CharacterOffsetBegin=579 CharacterOffsetEnd=580 PartOfSpeech=,] [Text=as CharacterOffsetBegin=581 CharacterOffsetEnd=583 PartOfSpeech=IN] [Text=did CharacterOffsetBegin=584 CharacterOffsetEnd=587 PartOfSpeech=VBD] [Text=French CharacterOffsetBegin=588 CharacterOffsetEnd=594 PartOfSpeech=JJ] [Text=Connection CharacterOffsetBegin=595 CharacterOffsetEnd=605 PartOfSpeech=NN] [Text=-RRB- CharacterOffsetBegin=605 CharacterOffsetEnd=606 PartOfSpeech=-RRB-] [Text=and CharacterOffsetBegin=607 CharacterOffsetEnd=610 PartOfSpeech=CC] [Text=gives CharacterOffsetBegin=611 CharacterOffsetEnd=616 PartOfSpeech=VBZ] [Text=you CharacterOffsetBegin=617 CharacterOffsetEnd=620 PartOfSpeech=PRP] [Text=some CharacterOffsetBegin=621 CharacterOffsetEnd=625 PartOfSpeech=DT] [Text=tough CharacterOffsetBegin=626 CharacterOffsetEnd=631 PartOfSpeech=JJ] [Text=characters CharacterOffsetBegin=632 CharacterOffsetEnd=642 PartOfSpeech=NNS] [Text=and CharacterOffsetBegin=643 CharacterOffsetEnd=646 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=647 CharacterOffsetEnd=648 PartOfSpeech=DT] [Text=glimpse CharacterOffsetBegin=649 CharacterOffsetEnd=656 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=657 CharacterOffsetEnd=659 PartOfSpeech=IN] [Text=life CharacterOffsetBegin=660 CharacterOffsetEnd=664 PartOfSpeech=NN] [Text=as CharacterOffsetBegin=665 CharacterOffsetEnd=667 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=668 CharacterOffsetEnd=669 PartOfSpeech=DT] [Text=cop CharacterOffsetBegin=670 CharacterOffsetEnd=673 PartOfSpeech=NN] [Text=at CharacterOffsetBegin=674 CharacterOffsetEnd=676 PartOfSpeech=IN] [Text=that CharacterOffsetBegin=677 CharacterOffsetEnd=681 PartOfSpeech=DT] [Text=time CharacterOffsetBegin=682 CharacterOffsetEnd=686 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=686 CharacterOffsetEnd=687 PartOfSpeech=.] 
root(ROOT-0, actor-6)
nn(S.-2, Roy-1)
nsubj(actor-6, S.-2)
cop(actor-6, was-3)
det(actor-6, a-4)
amod(actor-6, great-5)
prep(actor-6, from-7)
det(70-9, the-8)
poss(one-14, 70-9)
possessive(70-9, 's-10)
punct(Jaws-12, -LRB--11)
nsubj(one-14, Jaws-12)
cop(one-14, is-13)
pobj(from-7, one-14)
prep(one-14, of-15)
poss(favs-18, my-16)
amod(favs-18, all-time-17)
pobj(of-15, favs-18)
punct(favs-18, ,-19)
nn(Man-21, Marathon-20)
appos(favs-18, Man-21)
dep(Man-21, etc-22)
punct(favs-18, -RRB--23)
cc(actor-6, and-24)
mark(film-32, although-25)
det(7-ups-27, the-26)
nsubj(film-32, 7-ups-27)
cop(film-32, is-28)
neg(film-32, not-29)
det(film-32, an-30)
amod(film-32, Oscar-worthy-31)
advcl(puts-35, film-32)
punct(puts-35, ,-33)
nsubj(puts-35, it-34)
conj(actor-6, puts-35)
dobj(puts-35, you-36)
advmod(in-38, precisely-37)
prep(puts-35, in-38)
det(time-40, a-39)
pobj(in-38, time-40)
det(place-42, a-41)
dep(time-40, place-42)
punct(NY-44, -LRB--43)
appos(place-42, NY-44)
punct(NY-44, ,-45)
advmod(70-47, early-46)
dep(NY-44, 70-47)
possessive(70-47, 's-48)
punct(NY-44, ,-49)
dep(did-51, as-50)
dep(NY-44, did-51)
amod(Connection-53, French-52)
nsubj(did-51, Connection-53)
punct(NY-44, -RRB--54)
cc(puts-35, and-55)
conj(puts-35, gives-56)
iobj(gives-56, you-57)
det(characters-60, some-58)
amod(characters-60, tough-59)
dobj(gives-56, characters-60)
cc(characters-60, and-61)
det(glimpse-63, a-62)
conj(characters-60, glimpse-63)
prep(glimpse-63, of-64)
pobj(of-64, life-65)
prep(puts-35, as-66)
det(cop-68, a-67)
pobj(as-66, cop-68)
prep(puts-35, at-69)
det(time-71, that-70)
pobj(at-69, time-71)
punct(actor-6, .-72)

Sentence #5 (13 tokens):
And yes, the car chase is one of the all-time best.
[Text=And CharacterOffsetBegin=692 CharacterOffsetEnd=695 PartOfSpeech=CC] [Text=yes CharacterOffsetBegin=696 CharacterOffsetEnd=699 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=699 CharacterOffsetEnd=700 PartOfSpeech=,] [Text=the CharacterOffsetBegin=701 CharacterOffsetEnd=704 PartOfSpeech=DT] [Text=car CharacterOffsetBegin=705 CharacterOffsetEnd=708 PartOfSpeech=NN] [Text=chase CharacterOffsetBegin=709 CharacterOffsetEnd=714 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=715 CharacterOffsetEnd=717 PartOfSpeech=VBZ] [Text=one CharacterOffsetBegin=718 CharacterOffsetEnd=721 PartOfSpeech=CD] [Text=of CharacterOffsetBegin=722 CharacterOffsetEnd=724 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=725 CharacterOffsetEnd=728 PartOfSpeech=DT] [Text=all-time CharacterOffsetBegin=729 CharacterOffsetEnd=737 PartOfSpeech=JJ] [Text=best CharacterOffsetBegin=738 CharacterOffsetEnd=742 PartOfSpeech=JJS] [Text=. CharacterOffsetBegin=742 CharacterOffsetEnd=743 PartOfSpeech=.] 
root(ROOT-0, one-8)
cc(one-8, And-1)
advmod(one-8, yes-2)
punct(one-8, ,-3)
det(chase-6, the-4)
nn(chase-6, car-5)
nsubj(one-8, chase-6)
cop(one-8, is-7)
prep(one-8, of-9)
det(best-12, the-10)
amod(best-12, all-time-11)
pobj(of-9, best-12)
punct(one-8, .-13)

